cmd/celestia-appd/cmd/root.go

```diff
@@ -77,7 +78,13 @@ func NewRootCmd() *cobra.Command {
 				return err
 			}
 
+			// change tendermint config for celestia app
+			// set broadcast timeout to be 50 seconds in order to factor in long block times
+			// set max body bytes for transaction to be 8 Mb
+			// TODO: maybe we should make this timeout configurable?
```

Comment:
[follow-up issue]

> If my understanding is correct, timeouts aren't configurable through the cli

I think you're right based on no help message for it in:

```bash
celestia-appd help
```

<img width="1353" alt="Screen Shot 2022-10-20 at 9 39 30 AM" src="https://user-images.githubusercontent.com/3699047/196964297-058d0c07-b567-415e-a225-72aa02740b66.png">


```bash
$ celestia-appd start --rpc.max_body_bytes 300
Error: unknown flag: --rpc.max_body_bytes
```

but I think this would be a good follow-up issue especially if (as @SweeXordious points out) we have inconsistent block times so the 50 seconds may need to be modified.

Reply:
++ yeah this is new `good first issue`